frontend/javascripts/admin/voxelytics/workflow_view.tsx

```diff
@@ -339,11 +350,21 @@ export default function WorkflowView() {
         _report = selectMetaTask(_report, metatask);
       }
       setReport(_report);
+      setLoadingState({ status: "READY" });
     } catch (err) {
-      console.error(err);
-      Toast.error("Could not load workflow report.");
-    } finally {
-      setIsLoading(false);
+      try {
+        const organization =
+          user != null ? await isWorkflowAccessibleBySwitching(workflowName) : null;
+        setLoadingState({
+          status: "FAILED",
+          organizationToSwitchTo: organization ?? undefined,
+          error: err as Error,
+        });
+      } catch (err1) {
+        console.log(err1);
+        Toast.error("Could not load workflow report.");
+        setLoadingState({ status: "FAILED", error: err as Error });
```

Comment:
```suggestion
      } catch (accessibleBySwitchingError) {
        console.log(accessibleBySwitchingError);
        Toast.error("Could not load workflow report.");
        setLoadingState({ status: "FAILED", error: accessibleBySwitchingError as Error });
```

Reply:
note that this is not only renaming vars but also changes which err object is reported.